### ISSUE TITLE
Other Plugins dont work with workmanager

### ISSUE THREAD

**Issue by vivekcr** [vrtdev/flutter_workmanager#85]
After initialising the the workmanager and creating either of the tasks, If we use any plugins inside of the task execution it isn't recognised and throw an error as bellow
`MissingPluginException(No implementation found for method getLocation on channel lyokone/location)`

Actual code :
`Workmanager.executeTask((task, inputData) async {
    Location locationObject = Location();
    locationObject.getLocation();
    print(locationObject);
    return Future.value(true);
  }`

Basically any other plugin used inside of the task of work manager seems to be not recognised.

What am I missing, Do I need to register all my plugins again ?

**Comment by timrijckaert** [comment]
Can you provide a full sample that demonstrates this issue?
Which plugin are you using?
Did you follow the steps in the README?

**Comment by timrijckaert** [comment]
Never mind it seems like you are using [this plugin](https://github.com/Lyokone/flutterlocation/blob/master/android/src/main/java/com/lyokone/location/LocationPlugin.java#L118)
In the code you can see the plugin is not properly implemented and does not register itself when it has no Activity Context. 
You should open an issue on their Github.

**Comment by timrijckaert** [comment]
https://github.com/Lyokone/flutterlocation/issues/220
